Kernel/Output/HTML/Layout/LinkObject.pm

```diff
@@ -158,15 +158,15 @@ sub LinkObjectTableCreateComplex {
         next OBJECT if !$BackendObject;
 
         # get block data
-        my @BlockData = $BackendObject->TableCreateComplex(
+        my $BlockData = $BackendObject->TableCreateComplex(
```

Comment:
I think that @BlockData is correct, as there could be multiple hash references returned.